source/GameInterface/Services/MapEvents/PartyEnteredSettlement.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Common.Messaging;
+using ProtoBuf;
+
+namespace GameInterface.Services.MapEvents
+{
+    /// <summary>
+    /// Allow entry to a settlement.
+    /// </summary>
+    [ProtoContract(SkipConstructor = true)]
+    public record PartyEnteredSettlement : ICommand
```

Comment:
Commands should be current or future tense
For any network events, I've been prefixing with `Network`
```suggestion
    public record NetworkPartyEnterSettlement : ICommand
```